src/resolvers/profileResolver.ts

```diff
@@ -0,0 +1,53 @@
+import { Profile } from '../models/user'
+import cloudinary from 'cloudinary';
+import path from 'path';
+
+cloudinary.config({
+  cloud_name: process.env.CLOUD_NAME,
+  api_key: process.env.API_KEY,
+  api_secret: process.env.API_SECRET
+});
+
+const profileResolvers = {
+    Query: {
+        getProfile: (parent:any , args: any) => {
+            return Profile.findById(args._id)
+        }
+    },
+    Mutation: {
+        updateProfile: async (parent:any, args:any, context: any) => {
+            try {
+              let { _id, firstName, lastName, address, city, country, phoneNumber, biography, fileName } = args
```

Comment:
reduce the use of `let` to favor `const` as the ES6 standard to avoid giving large scope to the variables that will not be `mutated` afterwards